src/styles/tailwind.css

```diff
@@ -10,43 +10,43 @@ html {
 
 @layer base {
   /* Font families */
-  h1,
+  /* h1, */
   h2,
   h3,
   h4,
   h5 {
-    @apply font-display font-semibold;
+    @apply heading5;
```

Comment:
curious as to why we are setting heading5 for all h tags but then overriding them below. Should we just have a default for each? (ie. h1 = heading1, h2 = heading2, etc)